### PR TITLE
Row3,playboard

### PR DESCRIPTION
**Pull request by itsbluejelly**
Added favicon and meta tags in head section

### CODE REVIEW

File: docs/Tic-Tac-Toe.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Tech Titans">
+    <meta name="description" content="A multiplayer tic-tac-toe game">
+    <title>Tic-Tac-Toe-Toe by Tech Titans</title>
+    <!--Tic-Tac-Toe icon-->
+    <link rel="icon" href="./img/tic-tac-toe.png" type="image/x-icon">
```
Comment (Mwangangi1978) [inline docs/Tic-Tac-Toe.html:12]: great idea to download icon instead of linking .Love the icon

File: docs/Tic-Tac-Toe.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    </section>
+    <section>
+        <!--a section represents a row-->
+        <!--a span represents a single box-->
+        <span class="box7"></span>
+        <span class="box8"></span>
+        <span class="box9"></span>
+    </section>
+ </div>
+
```
Comment (Mwangangi1978) [inline docs/Tic-Tac-Toe.html:60]: I think we should add a footer to display our group name as authors as in the wire frames but we can also do this later if you wat to

**Review by Mwangangi1978 — APPROVED** [review]

File: docs/Tic-Tac-Toe.html (hunk 1 above)
Comment (itsbluejelly) [reply docs/Tic-Tac-Toe.html:12]: Thanks 👍🏿

File: docs/Tic-Tac-Toe.html (hunk 2 above)
Comment (itsbluejelly) [reply docs/Tic-Tac-Toe.html:60]: I think this is a great idea, sorry for the delay,let me see if I can update it😅😅

**Pull request merged**